app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -248,4 +258,10 @@ export default {
     padding: 0.2rem;
   }
 }
+
+.contact--mute {
+  color: #ff382d;
```

Comment:
Please use color variables from `dashboard/assets/scss/variables.scss`

Reply:
There is no variable for red other than `alert-color`. Should I use `alert-color` or add a new variable?

Reply:
Alert-color should be okay, we are working on standardizing the colours, so I would suggest using the variables which are already present.